app/forms/publish/course_study_site_form.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Publish
+  class CourseStudySiteForm < BaseCourseForm
+    FIELDS = %i[study_site_ids].freeze
```

Comment:
This seems like overkill. The class name is so specific that it would be weird to have any other fields here. It would necessitate renaming the class.

Reply:
We use this pattern in almost all of our forms on Publish (over 20), we can discuss alternative approaches after rollover when we have more time.